build-system/tasks/validator.js

```diff
@@ -33,6 +33,15 @@ async function validator() {
   });
 }
 
+/**
+ * Simple wrapper around the C++ validator tests
+ */
+async function validatorCppTest() {
+  execOrDie('bazel test validator_test', {
+    cwd: 'validator/cpp/engine',
```

Comment:
To be sure that the output is visible, add `stdio: 'inherit'` here.

Reply:
Done